core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -82,34 +91,44 @@ interface ActiveContributionDict {
 })
 
 export class TranslationSuggestionReviewModalComponent implements OnInit {
-  activeContribution: ActiveContributionDict;
-  activeContributionDetails: ActiveContributionDetailsDict;
-  authorName: string;
-  activeSuggestion: ActiveSuggestionDict;
-  activeSuggestionId: string;
+  // These properties are initialized using Angular lifecycle hooks
+  // and we need to do non-null assertion, for more information see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+  activeContribution!: ActiveContributionDict;
+  activeContributionDetails!: ActiveContributionDetailsDict;
+  authorName!: string;
+  activeSuggestion!: ActiveSuggestionDict;
+  activeSuggestionId!: string;
+  contentHtml!: string | string[];
+  editedContent!: EditedContentDict;
+  errorFound!: boolean;
+  errorMessage!: string;
+  explorationContentHtml!: string | string[];
+  finalCommitMessage!: string;
+  isCurriculumAdmin!: boolean;
```

Comment:
Let's start boolean variable names with non-verbs. E.g. userIsCurriculumAdmin, contentTypeIsHtml, contentTypeIsSetOfStrings, contentTypeIsUnicode

Reply:
Yes correct but i did not change that in this PR, their are lots of boolean variable names in this file which are wrong typed.
do i need to change all of them??

Reply:
Not edited any one, just use of `!:` and make them sorted.

Reply:
I would still suggest cleaning them up, if it's not too much trouble. We don't want to have to keep revisiting this file and fixing small things, it would be good to get it sorted out properly once and for all (especially given that the whole point of this project is to make things type-unambiguous).

Reply:
Done